.github/workflows/ios-unsigned.yml

```
@@ -112,26 +112,46 @@ jobs:
           echo "MODULE_CACHE_DIR=${MODULE_CACHE_DIR}" >> $GITHUB_ENV
           echo "CLANG_MODULE_CACHE_PATH=${MODULE_CACHE_DIR}" >> $GITHUB_ENV
 
+      - name: Detect available iOS Simulator runtime
+        id: detect_sim_runtime
+        run: |
+          set -euo pipefail
+          RUNTIME_ID="$(xcrun simctl list runtimes -j | /usr/bin/python3 - <<'PY'
+import json, sys
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 120: could not find expected ':'

(syntax-check)